### ISSUE TITLE
Cannot create new Instance of Aggregate with Keyword args on Ruby 3 

### ISSUE THREAD

**Issue by scottbartell** [RailsEventStore/rails_event_store#962]
If you create an instance of an aggregate that initializes with a keyword argument, like this:

```rb
class Customer
  include AggregateRoot

  def initialize(customer_uuid:); end
end
```

You get the following error on Ruby 3:
```
[3] DEVELOPMENT(main)> Customer.new(customer_uuid: SecureRandom.uuid)
ArgumentError: wrong number of arguments (given 1, expected 0; required keyword: customer_uuid)
from /Users/scott/src/ecommerce/app/domains/customer.rb:4:in `initialize'
```

I think the problem is here: https://github.com/RailsEventStore/rails_event_store/blob/master/aggregate_root/lib/aggregate_root.rb#L34

Which needs to be something like:
```
def new(*, **)
```

**Comment by mpraglowski** [comment]
Fix released in aggregate_root 2.0.1

**Comment by pawelpacana** [comment]
https://github.com/RailsEventStore/rails_event_store/releases/tag/v2.0.1
https://github.com/RailsEventStore/rails_event_store/releases/tag/v1.3.1
